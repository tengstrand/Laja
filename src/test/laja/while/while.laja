
#set (message = "")

#write "target/test-output/while.out"
  #while (message.length < 10)
    #set (message = "x{message}")
{message}
  #end
#end
