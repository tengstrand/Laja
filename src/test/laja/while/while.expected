x
xx
xxx
xxxx
xxxxx
xxxxxx
xxxxxxx
xxxxxxxx
xxxxxxxxx
xxxxxxxxxx
